File: code-coaches/easy/multiples/multiples.rb
def sum_multiples(num, *multiples)
  sum_of_multiples = (0...num).select { |x| multiples.any? { |m| x % m == 0 } }.sum
  return sum_of_multiples
end

def main()
  num = gets.chomp.to_i
  puts sum_multiples(num, 3, 5)
end

if __FILE__ == $0
  main()
end

File: code-coaches/easy/multiples/multiples.py
def sum_multiples(num: int, *multiples: int) -> int:
    sum_of_multiples = sum(x for x in range(num) if any(x % m == 0 for m in multiples))
    return sum_of_multiples


def main():
    num = int(input())
    print(sum_multiples(num, 3, 5))


if __name__ == "__main__":
    main()
